src/scenes/projects/projectCardStack.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { ProjectCard } from "./projectCard";
+import { StackInfo } from "../../data/stackIcons";
+import styles from "./projectCardStack.module.scss";
+
+interface ProjectCardStackProps {
+  stack: StackInfo[];
+}
+
+export function ProjectCardStack({ stack }: ProjectCardStackProps) {
+  return (
+    <div className={styles.container}>
+      <h5>Stack used:</h5>
+      <div>
+        {stack.map((icon) => (
+          <icon.component className={styles.stackIcon} />
```

Comment:
don't you get the missing key warning here?

Reply:
I do ;) I'll fix it on the next branch "smallFixes" until tomorrow.